lib/engines/content_object_store/app/services/content_object_store/create_edition_service.rb

```diff
@@ -32,6 +34,9 @@ def create_publishing_api_edition(content_id:)
 
     def publish_publishing_api_edition(content_id:)
       Services.publishing_api.publish(content_id)
+    rescue StandardError => e
```

Comment:
blocker: Rescuing from something so broad as `StandardError` gave me pause here because it would also catch anything we might be unexpectedly do wrong?

```ruby
Services.publishing_api.foo(content_id)
```

But I get the intention: we do want to be somewhat broad to catch any type of error with the integration 👍 

How about `GdsApi::HTTPErrorResponse` which [is defined here](https://github.com/alphagov/gds-api-adapters/blob/9b03c6d5e8ec03ce86d79fdc1ed0efdd0b474095/lib/gds_api/exceptions.rb#L25)?

(I now realise I used `GdsApi::HTTPUnprocessableEntity` in the other tests, perhaps that was realistic but too specific so we ought to change both over?)